Dockerfile

```diff
@@ -78,7 +93,7 @@ RUN npm install elm@0.18.0 \
     && wget "https://github.com/elm/compiler/releases/download/0.19.0/binaries-for-linux.tar.gz" \
     && tar xzf binaries-for-linux.tar.gz \
     && mv elm /usr/local/bin/elm19 \
-    && rm -f binaries-for-linux.tar.gz \
+    && rm -f binaries-for-linux.tar.gz
```

Comment:
Good catch - it's a minor miracle that this worked before!

Reply:
Ty, the IDE have detected it hahaha